packages/react-components/react-virtualizer/stories/Virtualizer/DefaultUnbounded.stories.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { Virtualizer, useStaticVirtualizerMeasure } from '@fluentui/react-components/unstable';
 import { makeStyles } from '@fluentui/react-components';
+// @ts-expect-error - we don support path aliases for libraries with both v8 and v9 as dependencies
```

Comment:
this needs to be here in order to pass `type-check` (which uses path aliases from tsconfig.base.json )

Reply:
wondering why the need of using v8 theme provider ?

Reply:
These Virtualizer examples intend to show that unbounded scrollviews in the body document, however due to the iFrame wrapper, these were being allowed to go to 100% the height with no overflow, while the iFrame container would provide scrolling.

I was using this to ensure the body's height was max 100VH, my mistake for using an old method, do we have a way to achieve this for the body in V9? I was not able to find a way to locally enforce 100VH on the body CSS in current storybook architecture.

Approving to land fix, will review the examples today and replace this if possible.